WordPress/Classes/ViewRelated/System/Notices/NoticePresenter.swift

```diff
@@ -296,14 +298,22 @@ class NoticePresenter: NSObject {
             }
 
             noticeContainer.noticeView.alpha = WPAlphaZero
-            noticeContainer.bottomConstraint?.constant = {
-                switch self.currentKeyboardPresentation {
-                case .present(let keyboardHeight):
-                    return -keyboardHeight + noticeContainer.bounds.height
-                case .notPresent:
-                    return self.window.untouchableViewController.offsetOffscreen
-                }
-            }()
+
+            switch self.currentNoticePresentation?.notice.style.animationStyle {
+            case .moveIn:
+                noticeContainer.bottomConstraint?.constant = {
+                    switch self.currentKeyboardPresentation {
+                    case .present(let keyboardHeight):
+                        return -keyboardHeight + noticeContainer.bounds.height
+                    case .notPresent:
+                        return self.window.untouchableViewController.offsetOffscreen
+                    }
+                }()
+            case .fade:
+                ()
+            case .none:
+                ()
```

Comment:
will these be implemented in other PRs?

Reply:
I changed this a bit to handle the optional value separately and added some notes in a comment.